packages/react/src/playground/index.css

```diff
@@ -14,4 +14,15 @@ body,
 
 section {
   margin-bottom: 16px;
+}
+
+.quick-links {
+  position: fixed;
```

Comment:
`position: absolute` seems to keep this contained within the SideNav area. Otherwise, 💯 .